platforms/documentation/docs/src/docs/userguide/authoring-builds/other/working_with_files.adoc

```diff
@@ -660,6 +660,26 @@ Default permissions differ for files and directories:
  * *file*: read & write for *owner*, read for *group*, read for *other* (*0644*, *rw-r--r--*)
  * *directory*: read, write & execute for *owner*, read & execute for *group*, read & execute for *other* (*0755*, *rwxr-xr-x*)
 
+[[sec:symlinks]]
+=== Dealing with symbolic links
+
+When copying files, Gradle will copy the contents of a symlink, rather than the symlink itself by default.
+If you want to copy the symlink as is, you can use the `linkStrategy` property on the `CopySpec` to control how symlinks are handled.
+The following example demonstrates how to copy all symlinks as symlinks:
+
+====
+include::sample[dir="snippets/files/copy/kotlin",files="build.gradle.kts[tags=symlinks]"]
+include::sample[dir="snippets/files/copy/groovy",files="build.gradle[tags=symlinks]"]
+====
+
+The available values for `linkStrategy` are:
+
+* `LinkStrategy.FOLLOW` (default): treat any symlink as a file it points to.
+* `LinkStrategy.ERROR`: fail the copy if a symlink is encountered.
+* `LinkStrategy.PRESERVE_ALL`: copy any symlink "as is".
+* `LinkStrategy.PRESERVE_RELATIVE` copy a symlink "as is" if it's relative and points to a file within copy root, otherwise fail the copy.
```

Comment:
```suggestion
* `LinkStrategy.PRESERVE_RELATIVE` copy a symlink "as is" if it's relative and points to a file within copy root; otherwise fail the copy.
```